manager/api/resources.go

```diff
@@ -15,6 +15,31 @@ type apiResponse interface {
 	empty() bool
 }
 
+// Identity
+type identityReq struct {
+	key string
+}
+
+type identityRes struct {
+	id string
+}
+
+func (res identityRes) headers() map[string]string {
+	return map[string]string{
+		"Location": res.id,
```

Comment:
`Location` should be used to provide resource URL, absolute or relative (in our case only the relative ones were used). I would recommend using a custom header for this, for example `X-client-id`.